.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,33 @@
+name: Static analysis
+on:
+  pull_request:
+
+jobs:
+  static_analysis:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '12.x'
```

Comment:
Why is java needed?

Reply:
seems like it's not needed, I removed it

Reply:
nice, my thoughts were that it'd be only needed for building the Android APK, but only for analyzing the code it would be unnecessary.... just to speed up things as much as possible